### ISSUE TITLE
Better dataset declaration / runtime parameter design

### ISSUE THREAD

**Issue by GeorgiChochov** [AbsaOSS/enceladus#33]
This involves the reduction of parameters submitted to the spark for both Standardization and Conformance. Instead these will be provided via Menas.
This should greatly improve the platform's integration capabilities, e.g. with  scheduling tools like Oozie.

**Comment by Zejnilovic** [comment]
A question, is this still in consideration? We do have convenience scripts now and @DzMakatun plans to do a "best spark options" provider

**Comment by GeorgiChochov** [comment]
Yes, this is for stuff like dataset `format` and additional format-specific options.
